project_scrum/project_scrum.py

```diff
@@ -0,0 +1,500 @@
+    task_count = fields.Integer(compute='_task_count')
+    task_test_count = fields.Integer(compute='_task_test_count')
+    review = fields.Html(string='Sprint Review', default="""
+        <h1 style="color:blue"><ul>What was the goal of this sprint?</ul></h1><br/><br/>
+        <h1 style="color:blue"><ul>Has the goal been reached?</ul></h1><br/><br/>
+    """)
+    retrospective = fields.Html(string='Sprint Retrospective', default="""
+        <h1 style="color:blue"><ul>What will you start doing in next sprint?</ul></h1><br/><br/>
+        <h1 style="color:blue"><ul>What will you stop doing in next sprint?</ul></h1><br/><br/>
+        <h1 style="color:blue"><ul>What will you continue doing in next sprint?</ul></h1><br/><br/>
+    """)
+    sequence = fields.Integer('Sequence', help="Gives the sequence order when displaying a list of tasks.")
+    progress = fields.Float(compute="_compute_progress", group_operator="avg", type='float', 
+                            multi="progress", string='Progress (0-100)', help="Computed as: Time Spent / Total Time.")
+    effective_hours = fields.Float(compute="_hours_get", multi="effective_hours", string='Effective hours', 
+                                   help="Computed using the sum of the task work done.")
+    planned_hours = fields.Float(multi="planned_hours", string='Planned Hours', 
+                                 help='Estimated time to do the task, usually set by the project manager when the task is in draft state.')
+    state = fields.Selection([('draft', 'Draft'), ('open', 'Open'), ('pending', 'Pending'), ('cancel', 'Cancelled'), ('done', 'Done')], 
+                             string='State', required=False)
+    company_id = fields.Many2one(related='project_id.analytic_account_id.company_id')
+
+    # Compute: effective_hours, total_hours, progress
+    @api.one
+    def _hours_get(self):
+        res = {}
+        effective_hours = 0.0
+        planned_hours = 0.0
+        for task in self.task_ids:
+            effective_hours += task.effective_hours or 0.0
+            planned_hours += task.planned_hours or 0.0
+        self.effective_hours = effective_hours
+        # self.planned_hours = planned_hours
+        return True
+
+    @api.onchange('project_id')
+    def onchange_project_id(self):
+        if self.project_id and self.project_id.manhours:
+            self.planned_hours = self.project_id.manhours
+        else:
+            self.planned_hours = 0.0
+
+    @api.onchange('date_start')
+    def onchange_date_start(self):
+        if self.date_start:
+            if self.project_id:
+                self.date_stop = fields.Date.from_string(self.date_start) +\
+                 timedelta(days=self.project_id.default_sprintduration)
+        else:
+            pass
+
+class project_user_stories(models.Model):
+    _name = 'project.scrum.us'
+    _description = 'Project Scrum Use Stories'
+    _order = 'reference'
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
```

Comment:
I'd suggest to also include https://github.com/OCA/server-tools/tree/10.0/base_kanban_stage for some advanced kanban behaviors and less code in your module. Can be extended by `base_kanban_stage_state`